modules/aggregation-layers/src/heatmap-layer/max-vs.glsl.js

```diff
@@ -6,5 +6,7 @@ void main()
 {
 outTexture = inTexture;
 gl_Position = vec4(0, 0, 0, 1.);
+// Enforce default value for ANGLE issue
```

Comment:
Link to https://bugs.chromium.org/p/angleproject/issues/detail?id=3236

Reply:
added issue that I opened, which links to original issue too.